src/main/python/utils/dynamo_s3_csv_exporter.py

```diff
@@ -0,0 +1,55 @@
+## Job to extract information from DynamoDB, write it into a CSV file and add it to S3.
+import csv
+import json
+import decimal
+import boto3
+from botocore.exceptions import NoCredentialsError
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, decimal.Decimal):
+            if o % 1 > 0:
+                return float(o)
+            return int(o)
+        return super(DecimalEncoder, self).default(o)
+
+def extract_and_transform():
+    dynamo_db = boto3.resource('dynamodb')
+    table = dynamodb.Table('survey-app-dump')
+
+    items = table.scan()['Items']
```

Comment:
we can check here if the items are empty here. Second we need a way to batch records and load. if there are millions of items, then scan all the records will not be possible. Hence, we can fetch a chunk write to file and fetch again. May be 128MB data scan at a time create a file on s3 then read next 128MB file2.csv on s3 so on.